src/styles/blocks/_product-card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  transition: transform 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+    &__title {
+    padding-top: 40px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+  }
+
+   &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 39px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    font-weight: bold;
+    font-size: 14px;
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &:hover .product-card__title {
+    color: #34569b;
+  }
+
+  nav a:hover {
+    color: #00acdc;
+  }
```

Comment:
1. you should use only classes for styling
2. those styles are for nav so add them to `nav__link`